x-pack/legacy/plugins/spaces/public/views/management/components/confirm_delete_modal.tsx

```diff
@@ -24,15 +24,13 @@ import {
   EuiText,
```

Comment:
Good call. I've been frustrated by this feature several times since I put it in. I'd like to change this to behave in a similar way to ESS, where we just ask the user to type 'DELETE' instead of the cluster name:

![image](https://user-images.githubusercontent.com/3493255/67945262-df446f80-fbb4-11e9-8903-c6b904ff5b42.png)


I'll open a separate issue for this, even though it'd be a rather small change.